Formula/terraform-provisioner-ansible.rb

```diff
@@ -0,0 +1,42 @@
+require "language/go"
+
+class TerraformProvisionerAnsible < Formula
+  desc "Ansible provisioner for Terraform"
+  homepage "https://github.com/jonmorehouse/terraform-provisioner-ansible"
+  url "https://github.com/jonmorehouse/terraform-provisioner-ansible/archive/0.0.2.tar.gz"
+  sha256 "5ba93ea5d6f28e156d7e4eba90cd89032f7dc6aff396a8429c903da4efa2d8b6"
+  head "https://github.com/jonmorehouse/terraform-provisioner-ansible.git"
+
+  depends_on "go" => :build
```

Comment:
Shouldn't this also have a dependency on terraform itself? (not for build like go, but for runtime)